.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
-sudo: false
+dist: xenial
 python:
     - 2.7
     - 3.4
     - 3.5
     - 3.6
-    - pypy
-    - pypy3
+    - 3.7
+    - pypy2.7-5.10.0
```

Comment:
It shouldn't be necessary to be explicit with PyPy version numbers, just 'pypy' and 'pypy3' should suffice to let them float. (Unless that's a xenial thing? It which case, echoing comments from others, we should probably restrict that `dist` to just the 3.7 case until Travis finishes ironing out the bugs in xenial).

Reply:
It's a xenial thing: https://github.com/travis-ci/travis-ci/issues/9452#issuecomment-438230628

Reply:
I understand your point on only using xenial for `3.7`. On the other hand, I would like to switch to the xenial system as much as possible, as this is the future. So I am undecided for now and torn to xenial.

Reply:
Let's keep using xenial only for Python 3.7 via `matrix`. When TravisCI once switches to xenial as default it does not hurt but the PyPy versions on xenial are quite old.

Reply:
I restricted xenial to `3.7` only.

Reply:
@mgedmin Interesting. I don't know why but I was believing `trusty` having 6.0 PyPy versions.

Reply:
Maybe it has pypy 6, but you have to ask for it?

That's the current situation on xenial: you can ask for pypy2.7-6.0 and pypy3.5-6.0, but if you ask for plain pypy3.5 you get 5.10.1 (and if you ask for pypy2 or pypy2.7, you get an error).